hack/verify-crdgen.sh

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env bash
+
+# Copyright 2021 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+SCHED_PLUGIN_ROOT=$(dirname "${BASH_SOURCE[0]}")/..
+source "${SCHED_PLUGIN_ROOT}/hack/lib/init.sh"
+
+kube::golang::verify_go_version
+
+cd "${SCHED_PLUGIN_ROOT}"
+
+CRD_OPTIONS="crd:trivialVersions=true,preserveUnknownFields=false"
+
+# Download controller-gen locally
+CONTROLLER_GEN="$(shell pwd)/bin/controller-gen"
+$(call go-get-tool,$(CONTROLLER_GEN),sigs.k8s.io/controller-tools/cmd/controller-gen)
```

Comment:
These 2 commands don't work on either my Mac or Linux env. Did you verify it works?

Reply:
Okay, I think I somehow configuring the `controller-gen` in the environment first that's why the commands might not be working. I'll update the code.

Reply:
This is still outstanding.

Reply:
I've updated the code and fixed the controller-gen command but facing the error as mentioned here: https://github.com/kubernetes-sigs/scheduler-plugins/pull/222#issuecomment-896978942